src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+import { Loader } from '../Loader/Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<string>('');
+  const [isLoaging, setIsLoaging] = useState<boolean>(false);
+  const [noPeople, setNoPeople] = useState(false);
```

Comment:
do we really need this state?🤔

```suggestion
  const noPeople = people.length === 0
```